bindings/go/src/ucx/memory.go

```diff
@@ -67,3 +72,12 @@ func (m *UcpMemory) Close() error {
 
 	return nil
 }
+
+func (mt *UcsMemoryType) Set (value string) error {
```

Comment:
does it mean that ucx go bindings set memory type by string instead of value?
IMO string->Value logic should be in perftest, not in go bindings.

Reply:
in golang interface can't be implemented on type from other package, though may created wrapper for mem type in perftest
on the other hand value -> string logic already implemented in go bindings, https://github.com/openucx/ucx/blob/7195d0473a38e329d0a1d831f37dc5ad3b1837da/bindings/go/src/ucx/ucs_constants.go#L33  and in those languages it's convenient to implement from/to string conversion along with the object